packages/lib/src/components/TablePagination/TablePagination.tsx

```diff
@@ -38,27 +42,45 @@ export function TablePagination<T>({
   setPageSize,
   setActivePage,
   handleRefetch,
+  fixedTablePageSize,
+  ...props
 }: Props<T>): JSX.Element {
   const { t } = useTranslation()
 
+  const handleSetFixedTablePageSize = useCallback(() => {
+    if (!fixedTablePageSize) return
+
+    table.setPageSize(fixedTablePageSize)
+
+    setPageSize(fixedTablePageSize)
+  }, [fixedTablePageSize, setPageSize, table])
+
+  useEffect(() => {
+    handleSetFixedTablePageSize()
+  }, [handleSetFixedTablePageSize])
```

Comment:
This can be summarized to:

```javascript
  useEffect(() => {
    if (!fixedTablePageSize) return

    table.setPageSize(fixedTablePageSize)

    setPageSize(fixedTablePageSize)
  }, [fixedTablePageSize, setPageSize, table])
```